draft-ietf-quic-invariants.md

```diff
@@ -316,6 +316,9 @@ connection IDs gives clients some assurance that the server received the packet
 and that the Version Negotiation packet was not generated by an off-path
 attacker.
 
+Version-specific rules for QUIC packets MUST NOT influence a server decision
```

Comment:
Are you saying that all QUIC servers need to send version negotiation packets in response to an 8-byte packet from an unknown version? That sounds like an amplification vector.

Reply:
The text in the transport doc is fine; this is, as David says, a little sketchy without the minimum length constraint.  A prefix of "other than the minimum packet size, ..." might work, I guess.

Reply:
Or scope the MUST NOT specifically to CID interpretation.

Reply:
Yes, I think this proposal is unclear on what the "MUST NOT" is for.